utilities/src/main/java/io/syndesis/qe/utils/DbUtils.java

```diff
@@ -42,6 +48,7 @@ public ResultSet executeSQLGetResultSet(String sqlCommand) {
             resultSet = preparedStatement.executeQuery();
         } catch (SQLException ex) {
             log.error("Error: " + ex);
+            //            fail("SQLException occurred");
```

Comment:
Don't we want to fail here?

Reply:
Not here. Because the first I/O exception would cause that the test suite failed. However, I forgot to delete this line.